autofit/non_linear/parallel/sneaky.py

```diff
@@ -183,6 +189,14 @@ def run(self):
         self.logger.debug("terminating process {}".format(self.name))
         self.job_queue.close()
 
+        if conf.instance["general"]["test"]["parallel_profile"]:
+
+            # TODO: RICH, is there a way for me to access the path of the search here?
+            # TODO: This is so I can put the sneaky_1234.prof files in a folder in the search path.
```

Comment:
Erm you could pass it in from abstract search -> SneakPool init -> SneakyProcess init

Reply:
BTW commenting on your own PR and tagging someone @Jammy2211 can be good for these questions